lint-staged.config.mjs

```diff
@@ -2,7 +2,7 @@ export default {
 	"*.rs": "rustfmt",
 	"*.{ts,tsx,js,mjs,yaml,yml}":
 		"node ./node_modules/prettier/bin/prettier.cjs --write",
-	"*.toml": "npx @taplo/cli format",
+	"*.toml": "pnpm @taplo/cli format",
```

Comment:
Should we use `pnpx` to replace `npx`?

<img width="2008" height="542" alt="image" src="https://github.com/user-attachments/assets/77f69d56-7c3e-4abd-8044-346540b39f34" />

```suggestion
	"*.toml": "pnpx @taplo/cli format",
```

Reply:
No need I think, `@taplo/cli` is already installed in devDependencies

Reply:
It does not work for me. Can you try it?

Reply:
Ahh you are right, didn't notice that